apm-lambda-extension/extension/process_env_test.go

```diff
@@ -172,9 +179,52 @@ func TestProcessEnv(t *testing.T) {
 		t.Fail()
 		return
 	}
-	config = ProcessEnv()
+	config = ProcessEnv(sm)
 	if config.LogLevel != zapcore.InfoLevel {
 		t.Log("Log level not set correctly")
 		t.Fail()
 	}
 }
+
+func TestGetSecretCalled(t *testing.T) {
```

Comment:
It seems that the Unit test, when run sequentially together with the whole test suite, fails due to previously set values of the `SECRET_TOKEN` and the `API_KEY`.
```suggestion
func TestGetSecretCalled(t *testing.T) {
    os.Clearenv()
```